anchor-counter-privy/client/src/index.css

```
@@ -0,0 +1,436 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background: #222;
+    font-family: 'Roboto', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)